MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -563,6 +554,31 @@
 			name = Frameworks;
 			sourceTree = "<group>";
 		};
+		C51923B41EA55C5E002AF9E1 /* Extensions */ = {
+			isa = PBXGroup;
+			children = (
+				351BEC081E5BCC72006FE110 /* Bundle.swift */,
+				351BEC0A1E5BCC72006FE110 /* Directions.swift */,
+				351BEC041E5BCC6C006FE110 /* ManeuverDirection.swift */,
+				351BEBEB1E5BCC63006FE110 /* String.swift */,
+				351BEBF01E5BCC63006FE110 /* UIView.swift */,
+			);
```

Comment:
MGLMapView+MGLNavigationAdditions.{h,m} is also an extension (or category, as it’s called in Objective-C).